machAeroTutorials/aero_adflow_polar.rst

```diff
@@ -42,9 +42,9 @@ Import libraries
 As in the previous example, we first import ADflow.
 We also need to import :doc:`baseclasses <baseclasses:index>`, which is a library of problem and solver classes used to encourage a common API within the MACH suite.
 We will again be using the AeroProblem, which is a container for the flow conditions that we want to analyze.
-However, in this case we will sequentially update the alpha parameter to produce the desired drag polar.
-Again, it is convenient to import the mpi4py library to prevent printing multiple times if we are running on multiple processors.
-Importing mpi4py is not entirely necessary in the runscript because ADflow does it internally if necessary.
+However, in this case we will sequentially update the :math:`\alpha` parameter to produce the desired drag polar.
+Again, it is convenient to import the ``mpi4py`` library to prevent printing multiple times if we are running on multiple processors.
+Importing ``mpi4py`` is not entirely necessary in the runscript because ADflow does it internally if necessary.
```

Comment:
Clarify with something like "ADflow does it internally if the scripts are invoked via mpirun" or something like that?